filament/src/Renderer.cpp

```diff
@@ -880,11 +882,16 @@ FrameGraphId<FrameGraphTexture> FRenderer::colorPass(FrameGraph& fg, const char*
 
                 // We set a "read" constraint on these attachments here because we need to preserve them
                 // when the color pass happens in several passes (e.g. with SSR)
+                auto depthAttachmentUsage = FrameGraphTexture::Usage::DEPTH_ATTACHMENT;
+                if (MTLPixelFormatDepth32Float_Stencil8 == config.depthFormat || MTLPixelFormatDepth24Unorm_Stencil8 == config.depthFormat) {
```

Comment:
we aren't in the Metal backend here, type must be `TextureFormat`
```suggestion
                if (TextureFormat::DEPTH32F_STENCIL8 == config.depthFormat || TextureFormat::DEPTH24_STENCIL8 == config.depthFormat) {
```

Reply:
fixed in https://github.com/shapr3d/filament/pull/58/commits/96d6ff120e6cb7e6eb5c5f37789eb0600043d643

Reply:
pretty much a nit because they're aliases, but filament internals use `TextureFormat` (especially this file), and `Texture::InternalFormat` is more for clients of the lib